### PR TITLE
RHOAIENG-16242: Updating YAML to include newly tested models

### PR DESCRIPTION
**Pull request by syaseen-rh**
## Description
Updating the YAML configuration for triton runtime to include newly tested models

## How Has This Been Tested?
Local build

## Preview [KServe-REST]
Screenshot 2024-12-04 at 12 04 30 PM

## Preview [KServe-GRPC]
Screenshot 2024-12-04 at 12 04 44 PM

## Preview [Modelmesh]
Screenshot 2024-12-04 at 12 08 50 PM

### CODE REVIEW

**Review by rpancham — APPROVED** [review]

File: modules/adding-a-tested-and-verified-runtime-for-the-multi-model-serving-platform.adoc [hunk 1]
```diff
@@ -63,7 +63,7 @@ spec:
           tritonserver "--model-repository=/models/_triton_models" "--model-control-mode=explicit" "--strict-model-config=false" "--strict-readiness=false" "--allow-http=true" "--allow-grpc=true"  '
       command:
         - /bin/sh
-      image: nvcr.io/nvidia/tritonserver:23.05-py3
+      image: nvcr.io/nvidia/tritonserver@sha256:1bde45b946f50f3e23a562cc5401d5ae7af8cb4a2daf037a55548241fadbd6f0
```
Comment (eturner24) [inline modules/adding-a-tested-and-verified-runtime-for-the-multi-model-serving-platform.adoc:66]: Isn't this an encryption key? Should this be in docs?

File: modules/adding-a-tested-and-verified-runtime-for-the-single-model-serving-platform.adoc [hunk 2]
```diff
@@ -48,7 +48,7 @@ spec:
         - --http-port=8080
         - --allow-grpc=true
         - --allow-http=true
-      image: nvcr.io/nvidia/tritonserver:23.05-py3
+      image: nvcr.io/nvidia/tritonserver@sha256:1bde45b946f50f3e23a562cc5401d5ae7af8cb4a2daf037a55548241fadbd6f0
```
Comment (eturner24) [inline modules/adding-a-tested-and-verified-runtime-for-the-single-model-serving-platform.adoc:51]: Same comment as previous, potentially sensitive info?

File: modules/adding-a-tested-and-verified-runtime-for-the-single-model-serving-platform.adoc [hunk 3]
```diff
@@ -102,7 +111,7 @@ spec:
...
```
Comment (eturner24) [inline modules/adding-a-tested-and-verified-runtime-for-the-single-model-serving-platform.adoc:114]: Same comment as previous

**Review by eturner24 — APPROVED** [review]

File: modules/adding-a-tested-and-verified-runtime-for-the-multi-model-serving-platform.adoc (hunk 1 above)
Comment (syaseen-rh) [reply modules/adding-a-tested-and-verified-runtime-for-the-multi-model-serving-platform.adoc:66]: @rpancham please advise.

**Pull request merged**